redhat-developer/app-services-cli
# fix(login): print info message when logging in through SSH

Closes #618 #617 

- 65ec2bc40b50ff9b55b06fff9de46f39b0118ab1 provides better explanations of the current limitations of the token-based login flow.
- def1b6c61cf00eaae23c907e20e824f467b284bd renames "MAS-SSO" occurrences to "identity.api.openshift.com"
- 6bbee309e92780f6e0d8195fcb3a2caad96f8421 I decided to NOT throw an error when there is an SSH session being used in conjunction with the browser-based login flow. This is because the developer can use X11 forwarding to forward to application display to their machine.

### Verification Steps

1. Run `ssh localhost`
2. Run `rhoas login`, you will see the following message:

```shell
SSH session detected: you may experience issues attempting to log in through a web browser. 
You can log in using an offline-token by passing the "--token" flag instead, which can be obtained at https://cloud.redhat.com/openshift/token.

Logging in...
```

### Type of change


### Checklist
